docs/extensions/readme.md

```diff
@@ -1192,6 +1193,143 @@ When set, specifies the header parameter to be used instead of `x-ms-client-requ
+
+**AllowedResource schema**:
+
+| Field Name  | Type       | Description                                                                                                                                                                                                         |
+| ----------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| scopes      | `[string]` | An array of scopes. See [Allowed Scopes](#allowed-scopes). If not specified, the default scope is `["ResourceGroup"]`.                                                                                              |
+| type        | `string`   | **Required** The type of resource that is being referred to. For example `Microsoft.Network/virtualNetworks` or `Microsoft.Network/virtualNetworks/subnets`. See [Example Types](#example-types) for more examples. |
+
+#### Allowed Scopes
+The following values are allowed for `scopes`. These values were derived from the [scope field in ARM templates](https://docs.microsoft.com/azure/azure-resource-manager/templates/scope-extension-resources?tabs=azure-cli).
+| Scope             | URL prefix                               | Meaning                                                                                                                                                                            |
+| ----------------- |------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Tenant`          | `/`                                      | The resource is deployed into a tenant                                                                                                                                             |
+| `Subscription`    | `/subscriptions`                         | The resource is deployed into a subscription                                                                                                                                       |
+| `ResourceGroup`   | `/subscriptions/resourceGroups`          | The resource is deployed into a resource group                                                                                                                                     |
+| `ManagementGroup` | `/Microsoft.Management/managementGroups` | The resource is deployed into a management group                                                                                                                                   |
+| `Extension`       | Any                                      | The resource is an extension resource and may be deployed as a subresource of another resource in any of the above scopes.                                                         |
```

Comment:
Do you intend to define the actual URL prefix? if so, they should be like following:

> Tenant
> `/`
> 
> Subscription
> `/subscriptions/{subscriptionId}/`
> 
> Resource Group
> `/subscriptions/{subscriptionId}/resourceGroups/{groupName}/`
> 
> Management Group
> `/providers/Microsoft.Management/managementGroups/{managementGroupName}/`
> 
> Extension
> `{parentScope}/providers/{extensionNamespace}/{extensionType}/{extensionName}/`
> 
>

Reply:
Fixed